### DISCUSSION TITLE
Store request data in local storage

### DISCUSSION THREAD

**Discussion by axelrindle** [scalar/scalar#2535] [Ideas]
I'd really love for `@scalar/api-reference` to store data entered in the modal request dialog to be saved in the local storage.

![modal](https://github.com/user-attachments/assets/fcb22bd3-8457-4ccc-acd3-5514aa1ff283)

**Comment by hanspagel** [comment]
Working on it 🏄

**Comment by sushovan-beemit** [comment]
Hi is there any update on this. Would really help our team if the request data is not lost on page refresh/reload

**Reply by sushovan-beemit** [reply]
hi @hanspagel apologies for tagging but this feature would be super super helpful for us. Is there any update on this.

**Reply by amritk** [reply]
Hey @sushovan-beemit sorry hans is OOO right now but I'll make a ticket out of this so you can track it

**Reply by amritk** [reply]
https://github.com/scalar/scalar/issues/5751
